cookbook_se/recipes/core/remote_flyte/README.rst

```diff
@@ -1,16 +1,18 @@
 Getting setup for remote execution
 -------------------------------------
-Flytekit provides a python SDK for authoring and executing workflows and tasks in python.
-Flytekit comes with a simplistic local scheduler that executes code in a local environment.
-But, to leverage the full power of Flyte, we recommend using a deployed backend of Flyte. Flyte can be run
-on a kubernetes cluster - locally, in a cloud environment or on-prem.
+Locally, Flytekit relies on the Python interpreter to execute both tasks and workflows.
+To leverage the full power of Flyte, we recommend using a deployed backend of Flyte. Flyte can be run
+on any Kubernetes cluster - a local cluster like `kind <https://kind.sigs.k8s.io/>`__, in a cloud environment or on-prem.
 
-Please refer to the `Installing Flyte <https://lyft.github.io/flyte/administrator/install/index.html>`_ for details on getting started with a Flyte installation.
-This section walks through steps on deploying your local workflow to a distributed Flyte environment, with ``NO CODE CHANGES``.
+Please refer to the `Installing Flyte <https://lyft.github.io/flyte/administrator/install/index.html>`__ for details on getting started with a Flyte installation.
+
+
+1. First commit your changes. Some of the steps below default to referencing the git sha.
+1. Run `make serialize_sandbox`. This will build the image tagged with just `flytecookbook:<sha>`, no registry will be prefixed. See the image building section below for additional information.
```

Comment:
are all the kinks with `serialize_sandbox` worked out?

Reply:
oh no, oops.  blah.  i guess i can delete for now.